api/v2/serializers/details/instance.py

```diff
@@ -60,7 +60,7 @@ def get_size(self, obj):
         return serializer.data
 
     def get_image(self, obj):
-        if obj.source.is_volume():
+        if not obj.source.is_machine():
```

Comment:
Does the commit message mean that the full fix is coming in another PR? Should this be a separate PR?